packages/flutter/lib/src/material/typography.dart

```diff
@@ -675,4 +703,91 @@ class Typography with Diagnosticable {
     labelMedium  : TextStyle(debugLabel: 'tall labelMedium 2018',  fontSize: 12.0, fontWeight: FontWeight.w400, textBaseline: TextBaseline.alphabetic),
     labelSmall  : TextStyle(debugLabel: 'tall labelSmall 2018',  fontSize: 11.0, fontWeight: FontWeight.w400, textBaseline: TextBaseline.alphabetic),
   );
+
+  /// Defines text geometry for [ScriptCategory.englishLike] scripts, such as
+  /// English, French, Russian, etc.
+  ///
+  /// The font sizes, weights, and letter spacings in this version match the
+  /// [2021 Material Design 3 specification](https://m3.material.io/styles/typography/overview).
+  static const TextTheme englishLike2021 =_M3Typography.englishLike;
+
+  /// Defines text geometry for dense scripts, such as Chinese, Japanese
+  /// and Korean.
+  ///
+  /// The Material Design 3 specification does not include 'dense' text themes,
+  /// so this is just here to be consistent with the API.
+  static const TextTheme dense2021 =_M3Typography.dense;
+
+  /// Defines text geometry for tall scripts, such as Farsi, Hindi, and Thai.
+  ///
+  /// The Material Design 3 specification does not include 'tall' text themes,
+  /// so this is just here to be consistent with the API.
+  static const TextTheme tall2021 =_M3Typography.tall;
```

Comment:
Nit: There should be a space after the `=`

Reply:
D'oh! Thx.